certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -362,7 +386,102 @@
         "summary": "Get terms by label"
       }
     },
-    "/workflows/09a7432e83914ef0a36d3219b0a28a44/triggers/manual/paths/invoke": {}
+    "/workflows/09a7432e83914ef0a36d3219b0a28a44/triggers/manual/paths/invoke": {},
```

Comment:
@Rens-Braspenning Empty operations can be removed as they would not add any value to your connector.